### ISSUE TITLE
custom training data allways very poor results.

### ISSUE THREAD

**Issue by cappittall** [googlesamples/mediapipe#250]
I have trained custom object detection model with relevant data (~750 annotated image x 3 augmantations) 
There are 4 choices for pretrained models. And tested 4 of them with different learning rate (default 0.3)
But I tried different combinations with 300 -200 epoch learning reate 0.01 to 0.3. However, the results always very very poor. Ap50 is max 0.25 the other values allways 

Here reproducted [result](https://colab.research.google.com/drive/19pwdw0FvMpzvqVtgIwyQOtP53Mnnk0sW?authuser=1#scrollTo=V5bIsWBZCb8d). 
What am I do wrong. ??? 

Note :
With same data I got ~86% at object detection with EfficientDet pretrained models as [desription](https://github.com/cappittall/pycoral_whl_4_python3.10/blob/main/tools/Object_Detection_with_EfficientDetTraining-%20PY3.9.ipynb).

**Comment by BoHellgren** [comment]
I have the same experience. I created a folder with 115 folders with cropped images of dogs and uploaded it to Google Drive. I ran the [MediaPipe Model Maker Image Classifier Demo](https://colab.research.google.com/github/googlesamples/mediapipe/blob/main/examples/customization/image_classifier.ipynb) with all defaults except for image_path and spec = image_classifier.SupportedModels (EfficientNet-Lite0). I expected to get a tflite model that is smaller, faster and more accurate when classifying dog images, compared to EfficientNet-Lite0. What I got is indeed smaller but the accuracy is lousy. I suspect this has to do with the metadata created by the model maker. If I display the metadata of the new model, it shows mean 0.0 std 255.0 while EfficientNet-Lite0 has mean 127.0 std 128.0.  See below for details. 
```
{
  "name": "ImageClassifier",
  "description": "Identify the most prominent object in the image from a known set of categories.",
  "subgraph_metadata": [
    {
      "input_tensor_metadata": [
        {
          "name": "image",
          "description": "Input image to be processed.",
          "content": {
            "content_properties_type": "ImageProperties",
            "content_properties": {
              "color_space": "RGB"
            }
          },
          "process_units": [
            {
              "options_type": "NormalizationOptions",
              "options": {
                "mean": [
                  0.0
                ],
                "std": [
                  255.0
                ]
              }
            }
          ],
          "stats": {
            "max": [
              1.0
            ],
            "min": [
              0.0
            ]
          }
        }
      ],
      "output_tensor_metadata": [
        {
          "name": "score",
          "description": "Score of the labels respectively.",
          "content": {
            "content_properties_type": "FeatureProperties",
            "content_properties": {
            }
          },
          "stats": {
            "max": [
              1.0
            ],
            "min": [
              0.0
            ]
          },
          "associated_files": [
            {
              "name": "labels.txt",
              "description": "Labels for categories that the model can recognize.",
              "type": "TENSOR_AXIS_LABELS"
            }
          ]
        }
      ]
    }
  ]
}

{
  "name": "EfficientNet-lite image classifier",
  "description": "Identify the most prominent object in the image from a set of 1,000 categories such as trees, animals, food, vehicles, person etc.",
  "version": "1",
  "subgraph_metadata": [
    {
      "input_tensor_metadata": [
        {
          "name": "image",
          "description": "Input image to be classified. The expected image is 224 x 224, with three channels (red, blue, and green) per pixel. Each element in the tensor is a value between min and max, where (per-channel) min is [-0.9921875] and max is [1.0].",
          "content": {
            "content_properties_type": "ImageProperties",
            "content_properties": {
              "color_space": "RGB"
            }
          },
          "process_units": [
            {
              "options_type": "NormalizationOptions",
              "options": {
                "mean": [
                  127.0
                ],
                "std": [
                  128.0
                ]
              }
            }
          ],
          "stats": {
            "max": [
              1.0
            ],
            "min": [
              -0.992188
            ]
          }
        }
      ],
      "output_tensor_metadata": [
        {
          "name": "probability",
          "description": "Probabilities of the 1000 labels respectively.",
          "content": {
            "content_properties_type": "FeatureProperties"
          },
          "stats": {
            "max": [
              1.0
            ],
            "min": [
              0.0
            ]
          },
          "associated_files": [
            {
              "name": "labels_without_background.txt",
              "description": "Labels for objects that the model can recognize.",
              "type": "TENSOR_AXIS_LABELS"
            }
          ]
        }
      ]
    }
  ],
  "author": "MediaPipe",
  "license": "Apache License. Version 2.0 http://www.apache.org/licenses/LICENSE-2.0."
}
